src/Interfaces/Action.ts

```diff
@@ -0,0 +1,4 @@
+export default interface Action {
+  type: string;
+  payload: any;
```

Comment:
Avoid using any, be specific

Reply:
@oesukam This is just a boilerplate to give an idea, but I will change it.